node/src/actors/chain_manager/superblock.rs

```diff
@@ -246,17 +256,17 @@ mod tests {
         };
 
         let expected_superblock = SuperBlock {
+            ars_length: 1,
             data_request_root: dr_merkle_root_1,
             tally_root: tally_merkle_root_1,
-            ars_root: PublicKeyHash::default().hash(),
+            ars_root: Hash::default(),
```

Comment:
you has a variable name called default_hash that you could use
```suggestion
            ars_root: default_hash,
```